drakrun/drakrun/storage.py

```diff
@@ -177,6 +180,7 @@ def vm0_root_as_block(self) -> Generator[str, None, None]:
 
         yield volume_path
 
+        subprocess.run(f"umount {volume_path}", shell=True)
```

Comment:
I don't think that this is the correct place to clean up the mount.
User of this function should be responsible for that. See `draksetup.py` - `create_rekall_profiles`.

The way it's implemented should unmount the partition but it probably mishandles it in case an exception happens.

Reply:
Exactly, that's why just to be safe, it is being unmounted here also, but not being checked for errors if the partition is already unmounted. Coz deleting without unmounting creates issues.